test/cppsuite/test_harness/workload/workload_tracking.h

```diff
@@ -89,15 +89,13 @@ class workload_tracking : public component {
         session = connection_manager::instance().create_session();
         testutil_check(
           session->create(session, _schema_table_name.c_str(), _schema_table_config.c_str()));
-        testutil_check(
-          session->open_cursor(session, _schema_table_name.c_str(), NULL, NULL, &_cursor_schema));
+        testutil_check(session->open_cursor(
+          session, _schema_table_name.c_str(), NULL, NULL, &_schema_track_cursor));
```

Comment:
For consistency, should we change NULL to nullptr here ? I see NULL if a few other places too..

Reply:
966e60f